src/App.jsx

```diff
@@ -13,7 +13,7 @@ function App() {
     <div className="App">
       <header className="App-header">
         <p>
-          Page has been open for <code>{count}</code> seconds.
+          spem bika  <code>{count}</code> seconds.
```

Comment:
S***? Seems like not safe for work, change text to some less _offensive_ version...

Reply:
Ok!